configdialog.cpp

```diff
@@ -578,6 +598,296 @@ void PaletteCustomizeDialog::get_and_save_color(int palette_index) {
 	}
 }
 
+std::wstring PaletteCustomizeDialog::export_palettes() {
+	auto all_palettes = m_palette_repo.get_all_custom_palettes();
+	std::wstring palettes_string = L"";
+
+	for (const auto &palette : all_palettes) {
+		auto name = palette.name;
+		auto *data = palette.data;
+
+		palettes_string.append(palette.name + L": " + m_palette_repo.serialize(*data) + L";\r\n");
```

Comment:
It's actually possible to create a palette with a colon in the name. This is obviously bad for the export string. There are two choices for resolving this:
1) Use a different separator than a `:` here
2) Just assume that nobody has made a palette with a colon in it, silently remove the colon from the list of valid characters when making a palette, and trust that nothing will go wrong?

Reply:
_Somehow_ I just never noticed that colons are valid characters. I think I'll go with option 1 even though it looks a bit worse imo.